### PR TITLE
fix slider displays uncorrectly, rearrange seekbar props

### PR DESCRIPTION
**Pull request by nyagami**
slider inside Seekbar component

### CODE REVIEW

**Comment by nyagami** [comment]
This fix will reduce number of rerendering of ReaderScreen from 12 to 9. 
Also screen will loading until fetch pre and next chapter

**Comment by nyagami** [comment]
Havent tested in special cases. But I think the seekbar can work quite well from this xD

File: src/screens/reader/ReaderScreen.js [hunk 1]
```diff
@@ -216,6 +204,18 @@ const ChapterContent = ({ route, navigation }) => {
     }
   }, []);
 
+  const [[nextChapter, prevChapter], setAdjacentChapter] = useState([]);
+
+  useEffect(() => {
+    const setPrevAndNextChap = async () => {
+      const nextChap = await getNextChapter(novelId, chapterId);
+      const prevChap = await getPrevChapter(novelId, chapterId);
```
Comment (rajarsheechatterjee) [inline src/screens/reader/ReaderScreen.js:212]: We could use `Promise.all()` for concurrency.

File: src/screens/reader/components/TextReader.tsx [hunk 2]
```diff
@@ -82,7 +82,7 @@ const TextReader: React.FC<TextReaderProps> = ({
               title={`Next: ${nextChapter.chapterName}`}
               onPress={navigateToNextChapter}
               theme={theme}
-              textColor={theme.colorButtonText}
+              textColor={theme.colorButtonText} // what is this prop ?
```
Comment (rajarsheechatterjee) [inline src/screens/reader/components/TextReader.tsx:85]: To override the button's text color.

Comment (nyagami) [reply src/screens/reader/components/TextReader.tsx:85]: but I didnt see colorButtonText in that interface

Comment (rajarsheechatterjee) [reply src/screens/reader/components/TextReader.tsx:85]: yeah, this was removed when themes were migrated to Material You, and I missed removing this.

**Review by rajarsheechatterjee — APPROVED** [review]

**Comment by rajarsheechatterjee** [comment]
Tested the build on emulator, the scroll position is being reset to top in `Render HTML` mode after scrolling to the actual position.

**Comment by nyagami** [comment]
I was quite confused about this thing. Maybe I should add (content size > 0) condition to doSaveProgress()

**Pull request merged**